### PR TITLE
Fixed AFL_FRIDA_INST_UNSTABLE_COVERAGE_FILE to work for instances that are  not named "default"

### PR DESCRIPTION
**Pull request by killerra**
Fixed this issue https://github.com/AFLplusplus/AFLplusplus/issues/2131

### CODE REVIEW

File: frida_mode/src/instrument/instrument_coverage.c [hunk 1]
```diff
@@ -818,6 +818,9 @@ void instrument_coverage_unstable_find_output(void) {
 
   GDir *dir = g_dir_open(fds_name, 0, NULL);
 
+  gchar *path_tmp = getenv("AFL_CUSTOM_INFO_OUT");
```
Comment (domenukk) [inline frida_mode/src/instrument/instrument_coverage.c:821]: You probably want to add AFL env variables to the list of supported envs in afl-fuzz, else afl-fuzz will complain

Comment (vanhauser-thc) [reply frida_mode/src/instrument/instrument_coverage.c:821]: No this is set by afl-fuzz

File: frida_mode/src/instrument/instrument_coverage.c [hunk 2]
```diff
@@ -829,7 +832,7 @@ void instrument_coverage_unstable_find_output(void) {
     if (link == NULL) { FFATAL("Failed to read link: %s", fullname); }
 
     gchar *basename = g_path_get_basename(link);
-    if (g_strcmp0(basename, "default") != 0) {
+    if (g_strcmp0(basename, instance_name) != 0) {
```
Comment (vanhauser-thc) [inline frida_mode/src/instrument/instrument_coverage.c:835]: instance_name is “out/foobar” not foobar

Comment (killerra) [reply frida_mode/src/instrument/instrument_coverage.c:835]: No, [gchar *instance_name = g_path_get_basename(path_tmp);](https://github.com/AFLplusplus/AFLplusplus/pull/2169/files#diff-8a67a75c5241f69cb74a414ec4ff56cab32dcead4104b3d7574638550df87a64R822) takes “out/foobar” from getenv("AFL_CUSTOM_INFO_OUT"); and returns "foobar"

**Comment by killerra** [comment]
However, there could be g_free() missing for the additional strings

**Comment by vanhauser-thc** [comment]
Thank you

**Comment by WorksButNotTested** [comment]
Awesome. Thanks @killerra , sorry I've had so little time to make fixes of late.

**Pull request merged**